MapboxMobileEvents/MMELocationManager.m

```diff
@@ -95,7 +96,7 @@ - (void)startLocationServices {
         if (authorizedAlways && self.hostAppHasBackgroundCapability) {
             [self.locationManager startMonitoringSignificantLocationChanges];
             [self startBackgroundTimeoutTimer];
-            if (@available(iOS 9.0, *)) {
+            if ([self respondsToSelector:@selector(allowsBackgroundLocationUpdates)]) {
```

Comment:
Great! Although, I think we still need to ignore the warning here, right?

Reply:
Ah, you're right. Missed that--thanks!